mirgecom/initializers.py

```diff
@@ -840,3 +841,149 @@ def __call__(self, x_vec, eos, *, t=0.0):
 
         return join_conserved(dim=self._dim, mass=mass, energy=energy,
                               momentum=mom, species_mass=specmass)
+
+
+class Discontinuity:
```

Comment:
Is there a more descriptive name we could use? `PlanarDiscontinuity`? `PlanarShock`?

Reply:
PlanarDiscontinuity would be the most descriptive, It's not guaranteed to be a shock, depending on the left and right states.